### PR TITLE
Feike/promscale

### PR DESCRIPTION
**Pull request by feikesteenbergen**
# Include promscale_extension in the Docker Image
    
The timescale_prometheus_extra is renamed to the promscale_extension.
For now, we include both versions.

# Reduce size of Docker Image

These directories are only required during building the Rust
componenents.

### CODE REVIEW

**Review by sgichohi — APPROVED** [review]

File: Dockerfile [hunk 1]
```diff
@@ -170,6 +171,23 @@ RUN if [ ! -z "${TIMESCALE_PROMETHEUS}" ]; then \
         done; \
     fi
 
+ARG TIMESCALE_PROMSCALE_EXTENSION=
+# build and install the promscale_extension extension
+RUN if [ ! -z "${TIMESCALE_PROMETHEUS}" ]; then \
```
Comment (cevian) [inline Dockerfile:176]:
```suggestion
RUN if [ ! -z "${TIMESCALE_PROMSCALE_EXTENSION}" ]; then \
```

File: Makefile [hunk 2]
```diff
@@ -34,6 +34,7 @@ TIMESCALEDB_RELEASE_URL?=$(TIMESCALEDB_IMAGE):$(TAG)
 TIMESCALEDB_LATEST_URL?=$(TIMESCALEDB_IMAGE):latest
 PG_PROMETHEUS?=
 TIMESCALE_PROMETHEUS?=0.1.0-beta.4
+TIMESCALE_PROMSCALE_EXTENSION?=cd691614
```
Comment (cevian) [inline Makefile:37]:
```suggestion
TIMESCALE_PROMSCALE_EXTENSION?=0.1.1
```

**Review by cevian — APPROVED** [review]

File: Dockerfile [hunk 3]
```diff
@@ -170,6 +171,23 @@ RUN if [ ! -z "${TIMESCALE_PROMETHEUS}" ]; then \
         done; \
     fi
 
+ARG TIMESCALE_PROMSCALE_EXTENSION=
+# build and install the promscale_extension extension
+RUN if [ ! -z "${TIMESCALE_PROMSCALE_EXTENSION}" ]; then \
+        curl https://sh.rustup.rs -sSf | bash -s -- -y \
```
Comment (sgichohi) [inline Dockerfile:177]: wonder if we could save time by not having to redo this now while the other extension lives

Comment (feikesteenbergen) [reply Dockerfile:177]: Takes only a few seconds, it does not redownload all required tools.

**Pull request merged**